lib/parse-imports.js

```diff
@@ -1,6 +1,22 @@
 var valueParser = require("postcss-value-parser")
 var stringify = valueParser.stringify
 
+function getMedia(params) {
+  var media = []
+  var last = params.reduce(function(medium, node, index) {
+    if (index < 2) {
+      return ""
+    }
+    if (node.type === "div" && node.value === ",") {
```

Comment:
The reason is
`(cond1 + cond2) * parentCond = cond1 * parentCond + cond2 * parentCond`